ynr/apps/elections/uk/management/commands/uk_create_elections_from_every_election.py

```diff
@@ -52,5 +65,7 @@ def handle(self, *args, **options):
             # be (re)set later
             Election.objects.update(current=False)
```

Comment:
In the case where this is invoked with `--poll-open-date` set, we're going to mark everything as not current, import all the elections for some given date (likely in the past) and then not mark any other elections as current (until the next scheduled sync). This is probably not what we want.